src/utils/actions.ts

```diff
@@ -24,11 +24,12 @@ export const stopForwarding = (action: FluxStandardAction<string, unknown, Actio
  * validateAction ensures that the action meets the right format and isn't scoped locally
  */
 export const validateAction = (
-    action: unknown
+    action: unknown,
+    blacklist: RegExp[] = []
```

Comment:
this should be an optional parameter. Also, it could be renamed as `denyList`.

Reply:
I renamed it but the parameter is automatically marked as optional because it has a default value.

```
Parameter cannot have question mark and initializer.ts(1015)
```